src/crimson/os/seastore/btree/fixed_kv_btree.h

```diff
@@ -592,6 +592,15 @@ class FixedKVBtree {
                 : true);
             }
           }
+        } else if (child == get_reserved_ptr()) {
+          if constexpr(
+            !std::is_base_of_v<typename internal_node_t::base_t,
+                               child_node_t>) {
+            assert(i->get_val().pladdr.is_paddr());
+            assert(i->get_val().pladdr.get_paddr() == P_ADDR_ZERO);
+          } else {
+            ceph_abort();
+          }
         }
       } else {
         ceph_abort("impossible");
```

Comment:
May be related questions:

Why `FixedKV[Internal/Leaf]Node::child_pos_for_key()` were implemented differently?

Why `check_node()` doesn't reuse `child_pos_for_key()`?

Reply:
> May be related questions:
> 
> Why `FixedKV[Internal/Leaf]Node::child_pos_for_key()` were implemented differently?
> 

I think I was modeling what `FixedKVBtree::lower_bound()` was doing. Seems that this is not necessary, will try to change it.

> Why `check_node()` doesn't reuse `child_pos_for_key()`?

Reply:
just fixed.

Reply:
All done.